src/compute/Cargo.toml

```diff
@@ -55,7 +55,7 @@ tower = { version = "0.4", features = ["util", "load-shed"] }
 tracing = "0.1"
 
 [target.'cfg(target_os = "linux")'.dependencies]
-tikv-jemalloc-ctl = { git = "https://github.com/risingwavelabs/jemallocator.git", rev = "b7f9f3" }
+tikv-jemalloc-ctl = { git = "https://github.com/risingwavelabs/jemallocator.git", rev = "64a2d9" }
```

Comment:
Also use the workspace dependency? 👀

Reply:
`tikv-jemalloc-ctl` is different from `tikv-jemallocator`, and it's only used in `compute` for memory controller policy. Are you sure to move it to workspace dependency as well?

Reply:
There can never exist two different version of jemalloc ctl
/allocator. So use worksapce makes sense to me.

Reply:
also it can be used in all nodes for profiling

Reply:
> `tikv-jemalloc-ctl` is different from `tikv-jemallocator`, and it's only used in `compute` for memory controller policy.

Oh, I didn't realize this.

Reply:
> also it can be used in all nodes for profiling

Agree. Let's do it upon that time!